deckhouse-controller/pkg/controller/module-controllers/release/controller.go

```diff
@@ -932,3 +1018,66 @@ type moduleValidator interface {
 	ValidateModule(m *addonmodules.BasicModule) error
 	GetValuesValidator() *validation.ValuesValidator
 }
+
+func (c *Controller) sendDocumentation(ctx context.Context, mr *v1alpha1.ModuleRelease) error {
+	addrs, err := c.getDocsBuilderAddresses(ctx)
+	if err != nil {
+		return fmt.Errorf("get docs builder addresses: %w", err)
+	}
+
+	if len(addrs) == 0 {
+		return nil
+	}
+
+	ms, err := c.moduleSourcesLister.Get(mr.GetModuleSource())
+	if err != nil {
+		return fmt.Errorf("get module source: %w", err)
+	}
+
+	md := downloader.NewModuleDownloader(c.externalModulesDir, ms, utils.GenerateRegistryOptions(ms))
```

Comment:
I'm wondering if we really have to download a release image one more time to get its documentation. We could get it from file system, couldn't we?

Reply:
There shouldn't be any problems with this, we can do it in the next release